qt_ui/windows/mission/flight/settings/QFlightStartType.py

```diff
@@ -26,6 +27,9 @@ def __init__(self, package_model: PackageModel, flight: Flight):
             self.start_type.addItem(start_type.value, start_type)
         self.start_type.setCurrentText(flight.start_type.value)
 
+        if isinstance(self.flight.departure, OffMapSpawn):
+            self.start_type.setEnabled(False)
+
```

Comment:
Disable the start-type combo-box if departure is from an "OffMapSpawn" CP

Reply:
Fixes the first exception in combination with the changes in `QFlightCreator.py`